### DISCUSSION TITLE
Standalone interface for DeDRM tools

### DISCUSSION THREAD

**Discussion by TheDcoder** [apprenticeharper/DeDRM_tools#1808] [General]
Hi, first of all, thanks for aggregating the tools and maintaining them!

Is there a standalone interface available at all? I quickly went through the source tree and didn't see anything of that sort. It would be nice if there was one available so that I can strip DRM without using Calibre (the process is a bit tedious there and I don't use it for any other purposes).

Assuming it doesn't exist, how hard would it be to implement a simple standalone command-line interface? Perhaps it could be done by spoofing parts of Calibre's plugin API :thinking:

**Comment by ElleKayEm** [comment]
There were standalone apps for Mac and Windows.  They were last included with release 6.6.3.  They used the same Python scripts the plugin does.

**Reply by TheDcoder** [reply]
That's awesome, thanks for sharing info, will look into them.

**Reply by ElleKayEm** [reply]
They've haven't been updated since whenever 6.6.3 was released.  You'll need Python 2, PyCrypto, and I'm not sure if there's anything else.  I find it easier to just use the calibre plugin myself.

**Reply by TheDcoder** [reply]
I understand, I will probably make a simple wrapper script to manually interact with the decryption scripts. I don't want to use Calibre as it messes up the output file name, and I don't really need it for other purposes.
